test/run-spec-wasm2c.py

```diff
@@ -173,6 +187,31 @@ def _CacheModulePrefixes(self):
                 self.module_prefix_map[name_idx] = name
                 self.unmangled_names[name_idx] = command['as']
 
+    def _WriteModuleInstanceImports(self, command, uninstantiable):
+        header_filename = utils.ChangeExt(command['filename'], '.h')
+        with open(os.path.join(self.out_dir, header_filename), encoding='utf-8') as f:
+            imported_modules = set()
+            for line in f:
+                if "import: " in line:
+                    line_split = line.split()
+                    import_module_name = MangleName(line_split[2][1:-1])
+                    imported_modules.add(import_module_name)
```

Comment:
It looks like the open file `f` is no longer needed after this line so everything below can/should be unintended perhaps?

Reply:
Indeed -- change made.